### PR TITLE
Improvement: Display all the taxonomy names in each posts in all the post layout

### PR DESCRIPTION
**Pull request by supriyajjadhav**
[Trello](https://trello.com/c/HG217qoX/214-fix-post-grid-masonry-carousel-taxonomy-issue)

### Description
If the block has more than 2 categories selected then in the post (Grid/masonry/carousel) layout only one category name is shown. We need to add all the selected taxonomy names in the posts

### Types of changes
- Added list of arrays to display all the categories in back-end and front-end.

### How has this been tested?
Checked whether it is working on the front-end and back-end.

### Checklist:
- [x] My code is tested
- [x] My code follows accessibility standards 
- [x] I've included any necessary tests - 
- [x] [ ] I've added proper labels to this pull request

### CODE REVIEW

File: dist/blocks/post/class-uagb-post.php [hunk 1]
```diff
@@ -1354,7 +1354,23 @@ public function render_meta_taxonomy( $attributes ) {
 			?>
 			<span class="uagb-post__taxonomy">
 				<span class="dashicons-tag dashicons"></span>
-				<?php echo esc_html( $terms[0]->name ); ?>
+				<?php
+				$a = count( $terms ) - 1;
+
+				foreach ( $terms as $key => $value ) {
+					// Get the ID of a given category
+					$category_id = get_cat_ID( $value->name );
```
Comment (vrundakansara) [inline dist/blocks/post/class-uagb-post.php:1362]: Please reuse the already existing ID from `$value->term_id`.

File: dist/blocks/post/class-uagb-post.php [hunk 2]
```diff
@@ -1354,7 +1354,23 @@ public function render_meta_taxonomy( $attributes ) {
...
+
+					// Get the URL of this category
+					$category_link = get_category_link( $category_id );
+					?>
+					<a href="<?php echo esc_url( $category_link ); ?>"><?php echo esc_html( $value->name ); ?></a>
```
Comment (vrundakansara) [inline dist/blocks/post/class-uagb-post.php:1367]: Push all these into an array variable and echo them by exploding the array.

**Review by vrundakansara — CHANGES_REQUESTED** [review]

File: dist/blocks/post/class-uagb-post.php (hunk 1 above)
Comment (supriyajjadhav) [reply dist/blocks/post/class-uagb-post.php]: Added $value->term_id in the function parameter

File: dist/blocks/post/class-uagb-post.php (hunk 2 above)
Comment (supriyajjadhav) [reply dist/blocks/post/class-uagb-post.php]: Created srray called $term_list and printed using implode

**Review by vrundakansara — APPROVED** [review]

**Review by AbhijitNage123 — APPROVED** [review]

**Review by vrundakansara — APPROVED** [review]

**Pull request merged**